build-tools/create-pkg/create-pkg.csproj

```diff
@@ -9,7 +9,7 @@
     <AssemblyName>create-pkg</AssemblyName>
     <OutputPath>..\..\bin\Build$(Configuration)</OutputPath>
   </PropertyGroup>
-  <Import Project="..\..\Configuration.props" />
```

Comment:
Don't we still need this import?

Reply:
This is imported though the `create-pkg.targets` project import, and we were being warned about multiple imports of this file during build. As such it has been removed in favor of the import though targets files.